learning-bitmark/working-with-bitmarks/creating-bitmark-account.md

```diff
@@ -0,0 +1,122 @@
+# Creating Bitmark account
+
+<br>
+<br>
+## Bitmark account number
+
+Property owners in Bitmark system are identified by their Ed25519 public keys. Those public keys are represented by the Bitmark account numbers which are in based58 endcoded format.
+
+* Account number - livenet example:
+
+    `bqSUHTVRYnrUPBEU48riv9UwDmdRnHm9Mf9LWYuYEa7JKtqgKw`
+
+* Account number - testnet example:
+
+    `fABCJxXc8aYGoj1yLLXmsGdWEo1Y5cZE9Ko5DrHhy4HvgGYMAu`
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark App
+
+* Download and install the [Android](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) or [iOS](https://apps.apple.com/us/app/bitmark-property-registry/id1429427796) Bitmark App
+
+* Open the app
+
+* Follow the instructions to create a new account
+
+* Check for the account number at Account > Settings screen
+
+* Backup the recovery phrase via Account > Settings screen > Write down recovery phrase
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark SDK
+
+In this section we are using the example of **Bitmark JS SDK** to create a new Bitmark account.<br>
+For further functions and other languages, please look at the [Bitmark SDK](https://github.com/bitmark-inc/docs/blob/master/bitmark-references/bitmark-sdk/bitmark-sdk-document.md) section.
+
+* Install Bitmark JS SDK
+
+    `npm install bitmark-sdk`
+
+* Initialize the SDK configuration
+
+    ```
+    const sdk = require('bitmark-sdk');
+
+    const config = {
+        API_token: "api-token",
+        network: "testnet"
+    };
+
+    sdk.init(config);
+    ```
+
+* Create a new Bitmark account:
+
+    `let account = new sdk.Account();`
+
+* Get the Account number:
+
+    `let account_number = account.getAccountNumber();`
+
+* Get the Account seed:
+
+    `let seed = account.getSeed();`
+
+* Get the Account recovery phrase:
+
+    `let phrase = account.getRecoveryPhrase();`
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark-CLI
+
+Following are the intructions to create Bitmark account on the testnet blockchain
+
+* Install [Bitmark-CLI](https://github.com/bitmark-inc/docs/blob/master/bitmark-references/bitmark-cli/bitmark-cli.md)
+
+* Create a new Bitmark account while initalizing the Bitmark-CLI configuration
+
+    `$bitmark-cli --network=testing --identity=first setup --connect=128.199.89.154:2130 --description='first user' --new`
+
```

Comment:
Another example of where explanation is definitely needed. What does "identity" signify? A name? Is that a location host that you have for the --connect?

Reply:
@shannona: In this section we just want to give users a quick start by trying copy & paste the commands or code. For the detailed explanation, we do have the separated documents for both Bitmark SDK & Bitmark-CLI. Which is mentioned and give links at the beginning of each section.